packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/gateway/GatewayRouter.sol

```diff
@@ -101,6 +103,34 @@ abstract contract GatewayRouter is TokenGateway {
             );
     }
 
+    function outboundTransferCustomRefund(
```

Comment:
we could add in a natspec comment saying that not all gateways support `outboundTransferCustomRefund` so this method may revert

Reply:
I don't think this solidity version allows us to inherit natspec comments, but we should also add in to the L1GatewayRouter function natspec

Reply:
added some natspec comments